src/jobs.rs

```diff
@@ -0,0 +1,23 @@
+use crate::{ban::unban_users, command_history::clear_command_history, SendSyncError, HOUR};
+use serenity::client::Context;
+use std::{
+    sync::atomic::{AtomicBool, Ordering},
+    thread::sleep,
+    time::Duration,
+};
+
+static JOBS_THREAD_INITIALIZED: AtomicBool = AtomicBool::new(false);
+
+pub(crate) fn start_jobs(cx: Context) {
+    if !JOBS_THREAD_INITIALIZED.load(Ordering::SeqCst) {
+        JOBS_THREAD_INITIALIZED.store(true, Ordering::SeqCst);
+        std::thread::spawn(move || -> Result<(), SendSyncError> {
```

Comment:
Why not spawn a Task? Would allow for async operations, and reusing a thread.

Reply:
Because right now we are not using an async runtime

Reply:
Oh... right, we were waiting for a Serenity update, which might have released? Haven't checked their change logs, but they are importing Tokio https://github.com/serenity-rs/serenity/blob/current/Cargo.toml#L80.